contribs/gmf/examples/style/mobileapp-vars.less

```diff
@@ -1,8 +1,13 @@
+@app-margin: 13px;
```

Comment:
It looks like this value is empirical and was determined to make it so the geolocation and + / - buttons are aligned, isn't it?